### DISCUSSION TITLE
max auto upload and delete files

### DISCUSSION THREAD

**Discussion by Fougl** [gopro/labs#888] [Feature Requests]
Hi, I know max does not have auto clear option after auto uploading to cloud. Is there a way to do it with labs qr?

**Comment by dnewman-gpsw** [comment]
No, sorry. No upload on MAX.  There is a clear command, but you will have to had done a manual backup first.

**Comment by Fougl** [comment]
Is the clear command or any command able to delete all gopro files from SD card?

I can only see this behaviour:
![image](https://github.com/gopro/labs/assets/21178929/2f1b6831-18ba-45cc-ac10-a86839407186)

Lets say my SD card is 16GB. Could I just put exactly 8GB of random files on it, so that the !C8 would then delete all the videos on it, if any were recorded?

**Reply by dnewman-gpsw** [reply]
The opposite, it would make sure you had 8GB clear, so no files would need to be deleted in your example.  If the card for full, 8GB of the oldest files would be deleted.
